File: 1635-number-of-good-pairs/solution.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        unordered_map<int,int> m;
        int count=0;
        for(auto ele : nums){
            count+=m[ele];
            m[ele] +=1;
        }
        return count;
    }
};

File: 1635-number-of-good-pairs/solution.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int cnt = 0;
        int n = nums.length;
        for(int i = 0;i<n-1;i++){
            for(int j = i+1;j<n;j++){
                if( i<j && nums[i] == nums[j]) cnt++;
            }
        }
        return cnt;
    }
}
